.coderabbit.yaml

```
@@ -0,0 +1,57 @@
+# Chat settings
+chat:
+  # Set chat language to Japanese
+  language: ja
+
+# Path filters (optional)
+# You can specify which files to include or exclude from reviews
+path_filters:
+  # Include all files by default
+  included_files:
+    - "**/*"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)